refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/test/java/de/muenchen/refarch/integration/dms/adapter/out/fabasoft/FabasoftAdapterTest.java

```diff
@@ -54,189 +55,197 @@
         this.properties.setUiUrl("uiurl");
-        val soapClient = FabasoftClienFactory.dmsWsClient("http://localhost:" + wmRuntimeInfo.getHttpPort() + "/");
+        final LHMBAI151700GIWSDSoap soapClient = FabasoftClienFactory.dmsWsClient("http://localhost:" + wmRuntimeInfo.getHttpPort() + "/");
         fabasoftAdapter = new FabasoftAdapter(properties, soapClient);
     }
 
     @Test
-    void execute_createFile_request() throws DmsException {
-        val response = new CreateFileGIResponse();
-        response.setObjid("1234567890");
+    void executeCreateFileRequest() throws DmsException {
+        final CreateFileGIResponse response = new CreateFileGIResponse();
+        response.setObjid(OBJID);
 
         WiremockWsdlUtility.stubOperation(
                 "CreateFileGI",
                 CreateFileGI.class, (u) -> "new file".equals(u.getShortname()),
                 response);
 
-        val file = new File("apentryCOO", "new file");
+        final File file = new File("apentryCOO", "new file");
 
-        val procedureResponse = fabasoftAdapter.createFile(file, "user");
+        final String procedureResponse = fabasoftAdapter.createFile(file, USER);
 
-        assertEquals(procedureResponse, "1234567890");
+        assertEquals(procedureResponse, OBJID);
     }
 
     @Test
-    void execute_createProcedure_request() throws DmsException {
-        val response = new CreateProcedureGIResponse();
-        response.setObjid("1234567890");
+    void executeCreateProcedureRequest() throws DmsException {
+        final CreateProcedureGIResponse response = new CreateProcedureGIResponse();
+        response.setObjid(OBJID);
 
         WiremockWsdlUtility.stubOperation(
                 "CreateProcedureGI",
                 CreateProcedureGI.class, (u) -> "new procedure".equals(u.getShortname()),
                 response);
 
-        val procedure = new Procedure("fileCOO", "new procedure", "custom file subject");
+        final Procedure procedure = new Procedure("fileCOO", "new procedure", "custom file subject");
 
-        val procedureResponse = fabasoftAdapter.createProcedure(procedure, "user");
+        final Procedure procedureResponse = fabasoftAdapter.createProcedure(procedure, USER);
 
-        assertEquals(procedureResponse.coo(), "1234567890");
+        assertEquals(procedureResponse.coo(), OBJID);
     }
 
     @Test
-    void execute_depositObject_request() throws DmsException {
-        val response = new DepositObjectGIResponse();
+    void executeDepositObjectRequest() throws DmsException {
+        final DepositObjectGIResponse response = new DepositObjectGIResponse();
         response.setObjid("objectCoo");
 
         WiremockWsdlUtility.stubOperation(
                 "DepositObjectGI",
                 DepositObjectGI.class, (u) -> true,
                 response);
 
-        fabasoftAdapter.depositObject("objectCoo", "user");
+        fabasoftAdapter.depositObject("objectCoo", USER);
     }
```

Comment:
Can we add some kind of assertion to test the successful case?

Reply:
used `assertDoesNotThrow` think that's a good fit.